cf-agent/verify_files.c

```diff
@@ -559,6 +583,74 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
 
 /*****************************************************************************/
 
+static PromiseResult WriteContentFromString(EvalContext *ctx, const char *path, const Attributes *attr,
+                                            const Promise *pp)
+{
+    assert(pp != NULL);
+    assert(attr != NULL);
+
+    PromiseResult result = PROMISE_RESULT_NOOP;
+
+    if (attr->content == NULL)
+    {
+        RecordFailure(ctx, pp, attr, "'content' not set for promiser: '%s'", pp->promiser);
```

Comment:
So this is the only `pp->promiser` reference now.

Reply:
And that's basically correct. The only problem is that it will show multiple failures (one for each file matched). So I'd still use `path` even here.

Reply:
Ok then.